pages/code-review.md

```diff
@@ -114,12 +149,26 @@ When reviewing code, you should be able to check off each of the following:
     - [ ] Do the changes address the project's needs?
     - [ ] Do the changes respect the project's existing style?
     - [ ] Does the new code avoid reproducing existing functionality?
-    - [ ] Are functions as simple as possible?
+    - [ ] Are functions/classes as simple as possible?
     - [ ] Is the code as efficient as possible?
-    - [ ] Is the usage of each function clear?
+    - [ ] Is the usage of each function/class clear?
     - [ ] Have edge cases been considered and tested for?
-
-Use this bookmarklet code to add this checklist to your Pull Request comments: [Code Review Checklist](https://gist.github.com/cfarm/a4174fe6f775353a3115)
+    - [ ] Does the code represent a logical unit of work?
+    - [ ] Are there any glaring syntax errors that were missed?
+    - [ ] Are language constructs being utilized properly?
+    - [ ] Are there no unecessary premature optimizations?
```

Comment:
Yeah, these seem like two-sides of the same coin to me now that you mention it.  On the one-hand code that is going to be merged should be efficient, but it also shouldn't go out of its way to address things that are unnecessary.  There are times when developers (myself included!) try and account for things in advance of them happening; defensive programming, so-to-speak.

On the surface these kinds of changes or improvements (in the eyes of the developer) look innocuous but in reality they could actually make things more difficult to refactor or extend later on, and I wanted to call this out explicitly.  I suppose this particular point could also fall under addressing scope creep, too.  I see how the bullet you mentioned is in conflict with this though, so I will attempt to re-word or just remove it.